### DISCUSSION TITLE
Any time soon to bring back the multimodal support for server?

### DISCUSSION THREAD

**Discussion by victorx98** [ggml-org/llama.cpp#7183] [General]
since it was removed in March.

**Comment by ggerganov** [comment]
Mostly depends on help from the community. Requires good understanding of `server` and updating the `llava` / `clip` APIs to allow batched decoding support, proper image prompt handling, improved memory management of image buffers, etc.

**Comment by domasofan** [comment]
That would be great if someone could have a look at it.
Blind people would profit a lot with llava and the web ui.

Greetings,
Simon
